grommet-leaflet/package.json

```diff
@@ -23,20 +23,26 @@
     "@types/leaflet.markercluster": "^1.5.2",
     "@types/react": "^18.2.23",
     "esbuild": "^0.19.4",
-    "prop-types": "^15.8.1"
+    "grommet": "^2.33.2",
+    "styled-components": "^5.1.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "dependencies": {
     "@react-leaflet/core": "^2.1.0",
-    "grommet": "^2.33.2",
     "grommet-icons": "^4.11.0",
     "leaflet": "^1.9.4",
     "leaflet.markercluster": "^1.5.3",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
     "react-leaflet": "^4.2.1",
-    "styled-components": "^5.1.0",
+    "prop-types": "^15.8.1",
     "uuid": "^9.0.1"
   },
+  "peerDependencies": {
+    "grommet": "^2.33.2",
```

Comment:
Does it have to be this version of grommet? or are any earlier versions supported?

Reply:
Good call out, we should be supporting earlier versions as well. Just not quite sure how far to go back. I checked and the components used in grommet-leaflet are pretty basic like Box, Button, and Text. So I think we could just say `"grommet": "^2.0.0"`.

Reply:
Cool yeah was just curious about that. Nothing wrong with encouraging a more recent version but wasn't sure if there was anything prohibiting an earlier one. I'm good with 2.0.0 and we can adjust if necessary.

Reply:
Just pushed up that change